src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/packagedeclaration/InputPackageDeclarationDiffDirectoryAtParent.java

```diff
@@ -1,8 +1,8 @@
 package com.puppycrawl.tools.checkstyle.checks.coding;
 
-class InputPackageDeclaration {
+public class InputPackageDeclarationDiffDirectoryAtParent {
```

Comment:
in future we should add public to input, as we already split inputs, so we do not expect that smb else require code in this form.